onadata/apps/api/viewsets/open_data_viewset.py

```diff
@@ -288,12 +290,30 @@ def destroy(self, request, *args, **kwargs):
         self.get_object().delete()
         return Response(status=status.HTTP_204_NO_CONTENT)
 
+    def _get_tableau_headers(self, xform):
+        """
+        Return a list of headers for tableau.
+        """
+        def shorten(xpath):
+            xpath_list = xpath.split('/')
+            return '/'.join(xpath_list[2:])
+
+        header_list = [
+            shorten(xpath) for xpath in xform.xpaths(
+                repeat_iterations=1)]
+        header_list += [
+            ID, UUID, SUBMISSION_TIME, TAGS, NOTES, REVIEW_STATUS,
+            REVIEW_COMMENT, VERSION, DURATION, SUBMITTED_BY, TOTAL_MEDIA,
+            MEDIA_COUNT, MEDIA_ALL_RECEIVED
+        ]
+        return header_list
+
     @action(methods=['GET'], detail=True)
     def schema(self, request, **kwargs):
         self.object = self.get_object()
         if isinstance(self.object.content_object, XForm):
             xform = self.object.content_object
-            headers = xform.get_headers()
```

Comment:
Why can't we use the headers returned by the `get_headers` function in tableau?

Reply:
> Why can't we use the headers returned by the get_headers function in tableau?

The repeat iteration value for nested repeats is static.
I did not mean to change that as it is used in defining column headers for csv exports, google sheets.